nx.json

```diff
@@ -12,7 +12,7 @@
   },
   "tasksRunnerOptions": {
     "default": {
-      "runner": "@nrwl/nx-cloud",
+      "runner": "nx/tasks-runners/default",
```

Comment:
**cohre (blocking)**: let's pull this out as it's unrelated and could cause issues